src/main/java/org/openrewrite/java/testing/assertj/JUnitAssertArrayEqualsToAssertThat.java

```diff
@@ -32,7 +32,10 @@
 import java.util.List;
 
 public class JUnitAssertArrayEqualsToAssertThat extends Recipe {
-    private static final String JUNIT_QUALIFIED_ASSERTIONS_CLASS_NAME = "org.junit.jupiter.api.Assertions";
+
+    private static final String JUNIT = "org.junit.jupiter.api.Assertions";
+    private static final String ASSERTJ = "org.assertj.core.api.Assertions";
+    private static final MethodMatcher ASSERT_ARRAY_EQUALS_MATCHER = new MethodMatcher(JUNIT + " assertArrayEquals(..)", true);
 
     @Override
     public String getDisplayName() {
```

Comment:
Are you seeing these warnings in your IDE as well?
![image](https://github.com/user-attachments/assets/e7a58a93-e192-4e8d-824d-ed56f970f269)

As a general rule we try to completely remove warnings on recipes we update, such that there's no nagging about warnings when we go to commit (enable that same check in your IDE). Would be a good moment to clear these warnings out as well by using a lower case `t` on `To`.

Reply:
Yeah, I saw it too. I will go scroll through the code, in case I missed more of those.